node_vwf.js

```diff
@@ -152,8 +152,8 @@ function startVWF() {
         'destroy buffer size': Infinity, // https://github.com/socketio/socket.io/issues/1592#issuecomment-45885463; should be fixed in socket.io 1.0
     } );
     socketManager.set( 'transports', [ 'websocket' ] );
-    socketManager.set( 'heartbeat timeout', 600 );
-    socketManager.set( 'close timeout', 60 );
+    socketManager.set( 'heartbeat timeout', 3600 ); // ! hour timeout
+    socketManager.set( 'close timeout', 3600 ); // 1 hour timeout
```

Comment:
The close timeout should be left at 60. This is the default, so this line could be removed.

The close timeout measures the interval that the server waits after a client has disconnected unexpectedly until it gives up. The server records messages that the client has missed during this interval, so the timeout shouldn't be any longer than necessary.

Reply:
This is what has already been delivered in TDK 1.1, so any further tweaking to the timeout parameters will have to be done in a new branch.

Reply:
A new task has been added to the backlog to continue testing periodically at scale and tuning as necessary.  In the meantime, this change will be merged back into the mainline to preserve what's been delivered historically.